app/forms/routing/routing_plan_static_route_batch_creator_form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Routing::RoutingPlanStaticRouteBatchCreatorForm < ApplicationForm
+  def self.inheritance_column
+    :_type
+  end
```

Comment:
method can be removed

when `inheritance_column` return non-existing column name it same as it return `nil`